store/sqlstore/retention_policy_store.go

```diff
@@ -586,70 +727,117 @@ func (s *SqlRetentionPolicyStore) DeleteOrphanedRows(limit int) (deleted int64,
-	SELECT COUNT(*)
-	FROM Users
-	INNER JOIN TeamMembers ON Users.Id = TeamMembers.UserId
-	INNER JOIN Teams ON TeamMembers.TeamId = Teams.Id
-	INNER JOIN RetentionPoliciesTeams ON Teams.Id = RetentionPoliciesTeams.TeamId
-	INNER JOIN RetentionPolicies ON RetentionPoliciesTeams.PolicyId = RetentionPolicies.Id
-	WHERE Users.Id = :UserId
-		AND TeamMembers.DeleteAt = 0
-		AND Teams.DeleteAt = 0`
-	props := map[string]interface{}{"UserId": userID}
-	return s.GetReplica().SelectInt(query, props)
+	query := s.getQueryBuilder().
+		Select("Count(*)").
+		From("Users").
+		InnerJoin("TeamMembers ON Users.Id = TeamMembers.UserId").
+		InnerJoin("Teams ON TeamMembers.TeamId = Teams.Id").
+		InnerJoin("RetentionPoliciesTeams ON Teams.Id = RetentionPoliciesTeams.TeamId").
+		InnerJoin("RetentionPolicies ON RetentionPoliciesTeams.PolicyId = RetentionPolicies.Id").
+		Where(
+			sq.And{
+				sq.Eq{"Users.Id": userID},
+				sq.Eq{"TeamMembers.DeleteAt": int(0)},
+				sq.Eq{"Teams.DeleteAt": int(0)},
+			},
+		)
+
+	queryString, args, err := query.ToSql()
+	if err != nil {
+		return 0, errors.Wrap(err, "team_policies_count_for_user_tosql")
+	}
+
+	var count int64
+	if err := s.GetReplicaX().Get(&count, queryString, args...); err != nil {
+		return 0, errors.Wrap(err, "failed to count TeamPoliciesCountForUser")
+	}
+
+	return count, nil
 }
 
 func (s *SqlRetentionPolicyStore) GetChannelPoliciesForUser(userID string, offset, limit int) (policies []*model.RetentionPolicyForChannel, err error) {
-	const query = `
-	SELECT Channels.Id, RetentionPolicies.PostDuration
-	FROM Users
-	INNER JOIN ChannelMembers ON Users.Id = ChannelMembers.UserId
-	INNER JOIN Channels ON ChannelMembers.ChannelId = Channels.Id
-	INNER JOIN RetentionPoliciesChannels ON Channels.Id = RetentionPoliciesChannels.ChannelId
-	INNER JOIN RetentionPolicies ON RetentionPoliciesChannels.PolicyId = RetentionPolicies.Id
-	WHERE Users.Id = :UserId
-		AND Channels.DeleteAt = 0
-	ORDER BY Channels.Id
-	LIMIT :Limit
-	OFFSET :Offset`
-	props := map[string]interface{}{"UserId": userID, "Limit": limit, "Offset": offset}
-	_, err = s.GetReplica().Select(&policies, query, props)
+	query := s.getQueryBuilder().
+		Select(`Channels.Id as "Id", RetentionPolicies.PostDuration`).
+		From("Users").
+		InnerJoin("ChannelMembers ON Users.Id = ChannelMembers.UserId").
+		InnerJoin("Channels ON ChannelMembers.ChannelId = Channels.Id").
+		InnerJoin("RetentionPoliciesChannels ON Channels.Id = RetentionPoliciesChannels.ChannelId").
+		InnerJoin("RetentionPolicies ON RetentionPoliciesChannels.PolicyId = RetentionPolicies.Id").
+		Where(
+			sq.And{
+				sq.Eq{"Users.Id": userID},
+				sq.Eq{"Channels.DeleteAt": int(0)},
+			},
+		).OrderBy("Channels.Id").Limit(uint64(limit)).Offset(uint64(offset))
+
+	queryString, args, err := query.ToSql()
+	if err != nil {
+		return nil, errors.Wrap(err, "channel_policies_for_user_tosql")
+	}
+
+	if err := s.GetReplicaX().Select(&policies, queryString, args...); err != nil {
```

Comment:
`policies := []*model.RetentionPolicyForChannel{}` Let's remove the named returns for this.